data_labeling_examples/bulk_labeling_java/src/main/java/com/oracle/datalabelingservicesamples/labelingstrategies/FirstLetterMatch.java

```diff
@@ -11,7 +11,7 @@ public class FirstLetterMatch implements RuleBasedLabelingStrategy {
 	@Override
 	public List<String> getLabel(RecordSummary record) {
 		for (String label : Config.INSTANCE.getLabels()) {
-			if (record.getName().startsWith(label)) {
+			if (record.getName().startsWith(String.valueOf(label.charAt(0)))) {
```

Comment:
Why was this change required in the existing functionality?

Reply:
This change is related to this bug : https://jira.oci.oraclecorp.com/browse/DLS-4883